python/startup/bootstrap.py

```diff
@@ -37,20 +37,26 @@ def bootstrap_sgtk_classic():
         error("Shotgun: Could not import sgtk! Disabling for now: %s" % e)
         return
 
+    sgtk.LogManager().initialize_base_file_handler("tk-3dsmaxplus")
+    logger = sgtk.LogManager.get_logger(__name__)
+
     if not "TANK_ENGINE" in os.environ:
+        logger.error("Shotgun: Missing required environment variable TANK_ENGINE.")
```

Comment:
Nitpicking here, but you can probably ditch the "Shotgun:" at the head of the log messages going to our logger.